### ISSUE TITLE
False positive S4653: 'x' unit

### ISSUE THREAD

**Issue by vilchik-elena** [SonarSource/sonar-css#139]
```
.foo {
  background-image:
    image-set(
      url(../images/background-blended.jpg?size=400&quality=10) 1x,
      url(../images/background-blended.jpg?size=800&quality=10) 2x,
      url(../images/background-blended.jpg?size=1200&quality=10) 3x
    );
  background-position: top center;
  background-repeat: no-repeat;
}
```

PR is created on Stylelint side: https://github.com/stylelint/stylelint/pull/3789
When new version of Stylelint is released, we can update dependency, thus fixing current FP

**Comment by vilchik-elena** [comment]
devs of stylelint are not happy with too straightforward PR I've provided (stylelint/stylelint#3789). So we will go for the config inside sonar-css
